query-parser/src/parser_util.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! matches_plain_symbol {
 /// The provided `$body` will be evaluated with `$input` bound to the input stream.
 ///
 /// `$body`, when run, should return a `ParseResult` of the appropriate result type.
+#[macro_export]
```

Comment:
It was: they're very EDN-specific, and I wasn't sure how reusable they would be.

It looks like @joewalker could directly use `def_value_satisfy_parser_fn!`, though…